Appium/Dockerfile

```diff
@@ -83,7 +83,7 @@ ENV PATH=$PATH:$ANDROID_HOME/platform-tools:$ANDROID_HOME/build-tools
 #====================================
 # Install latest nodejs, npm, appium
 #====================================
-ARG APPIUM_VERSION=1.7.0-beta
+ARG APPIUM_VERSION=1.8.1
```

Comment:
We control Appium version from Travis environment variables so it's not necessary to update it here. Also '1.8.1-p2' is new image published on June. Please check our release tags for details.